f8a_worker/workers/graph_importer_utils.py

```diff
@@ -0,0 +1,183 @@
+import os
+import requests
+import logging
+import json
+import traceback
+from f8a_worker.workers.graph_populator import GraphPopulator
+from selinon import StoragePool
+
+logger = logging.getLogger(__name__)
+
+# Get Variable Parameters from Environment
+GREMLIN_SERVER_URL_REST = "http://{host}:{port}".format(
+                           host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST", "localhost"),
+                           port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT", "8182"))
+
+
+class GETS3:
```

Comment:
This class is sort of useless, why is it here?

Reply:
it is used

Reply:
yes, but like an object, this class does not make much sense to me.

Reply:
Is there a better way in your mind? Maybe I am not aware on how to get this handle in a better way.

Reply:
> Is there a better way in your mind? Maybe I am not aware on how to get this handle in a better way.

Just request these adapters in the code where you need them. You don't need to care about these much - JFYI they are connecting lazily and the connection is shared across the whole worker codebase to optimize access (part of worker "global" context). For a user using such adapter - just ask for it and use it's API.

Ideally we would like to introduce the same adapter for graph database as well so we can share implementation in other parts of project (e.g. server, ...). This will minimize code, simplify it and in case of bugs, there will be a straightforward fix at one place.

Reply:
This looks like outside the scope for this PR. Please go ahead and create a technical debt around this.